docs/contributor/06-module-migration-concept.md

```diff
@@ -0,0 +1,79 @@
+# New Module Submission and Promotion Concept
+
+The existing module metadata is in channel-based ModuleTemplate custom resources (CRs). The new module metadata sources are version-based ModuleTemplates accompanied by ModuleReleaseMeta CRs. This document describes the target module submission process for module development teams. Migrating a Kyma module from the existing module metadata to the new module metadata.
```

Comment:
The last sentence "Migrating a Kyma module from the existing module metadata to the new module metadata." reads a cut-off.